extensions/objects/templates/skill-selector-editor.directive.html

```diff
@@ -3,22 +3,21 @@
     Search By Skill
     <input type="text" class="form-control protractor-test-skill-name-input" placeholder="Enter a skill name" ng-model="skillFilterText" autofocus>
   </md-card>
-  <div style="max-height: 50vh; overflow-y: auto;">
-    <div class="list-group-item protractor-test-rte-skill-selector-item"
+  <div class="skill-selector-item-container">
+    <div class="oppia-list-group-item-text list-group-item protractor-test-rte-skill-selector-item"
          ng-repeat="skill in $ctrl.skills | filter:skillFilterText"
          ng-click="$ctrl.selectSkill(skill.id, skill.description)"
-         ng-class="{'list-group-item-success': (skill.id === $ctrl.value)}"
-         style="color: black;">
+         ng-class="{'list-group-item-success': (skill.id === $ctrl.value)}">
       <label class="list-group-item-heading"><[skill.description]></label>
-      <div ng-if="skill.id === $ctrl.value" style="color: black;">
+      <div ng-if="skill.id === $ctrl.value" class="oppia-skill-summary-text">
         <strong> Explanation </strong>
-        <div style="padding-bottom: 10px; padding-top: 5px;">
+        <div class="skill-explanation-container">
           <angular-html-bind html-data="skill.skill_contents.explanation.html">
           </angular-html-bind>
         </div>
         <strong> Worked Examples </strong>
         <ul>
-          <li ng-repeat="example in skill.skill_contents.worked_examples" style="padding-top: 5px;">
+          <li ng-repeat="example in skill.skill_contents.worked_examples" class="example-questions-container">
```

Comment:
Not just questions, explanations are also there in this, right?
Maybe just worked-example-container?

Reply:
Done.